sale_timesheet_task_exclude/tests/__init__.py

```diff
@@ -1,10 +1,3 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
-from odoo.addons.sale_timesheet.tests import common
```

Comment:
If this module changes behaviour, it should not affect other module tests

Reply:
On top of that - current v12 branch passes the tests, so please adjust your PR not to break anything

Reply:
@alexey-pelykh I'm not sure not breaking odoo's behaviour is a "good reason" to load these tests, but anyway it helped identifying an issue we had in the computation that we wouldn't have find without. So I reverted the commit, and fixed the issue. :wink:

Reply:
See! It worked out with benefits!